src/main/java/com/mongodb/jdbc/utils/X509Authentication.java

```diff
@@ -16,19 +16,26 @@
 
 package com.mongodb.jdbc.utils;
 
+import com.mongodb.MongoException;
```

Comment:
We have an [integration test](https://github.com/mongodb/mongo-jdbc-driver/blob/7dbaf15e0b93b55fab0c3f76a6d0ec32ada38b4c/src/integration-test/java/com/mongodb/jdbc/integration/AuthX509IntegrationTest.java) for the PKCS#8 type PEM file.  Just in case, how about we create a ticket to add a test for confirming the PKCS#1 type PEM file? It can be something backlogged for if it is necessary to test.